client/stylesheets/_modal.scss

```diff
@@ -371,6 +371,16 @@
 	margin: 8px 0 15px 0 !important;
 }
 
+.profileReceivedPriceOriginal {
```

Comment:
Selector `profileReceivedPriceOriginal` should be written in lowercase with hyphens